clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3423,12 +3425,16 @@ public void consumerBatchUnsubscribe(Set<PubSubTopicPartition> topicPartitionSet
   public abstract void consumerUnSubscribeAllTopics(PartitionConsumptionState partitionConsumptionState);
 
   public void consumerSubscribe(PubSubTopicPartition pubSubTopicPartition, long startOffset, String kafkaURL) {
-    final boolean consumeRemotely = !Objects.equals(kafkaURL, localKafkaServer);
+    // TODO: pass special format of kafka url as input here when subscribe to the separate incremental push topic
+    String resolvedKafkaURL = kafkaClusterUrlResolver != null ? kafkaClusterUrlResolver.apply(kafkaURL) : kafkaURL;
```

Comment:
Just to confirm for future work: this is the remaining work we will need to change in the next phase of the implementation right? Because as I see it is still subscribing to original context.

Reply:
That's right. When SIT needs to subscribe sep topic, it should pass the kafka topic that has "_sep" suffix to this method such that the corresponding `KafkaConsumerServic` and `TopicManager` will be used and the `StorePartitionDataReceiver` will also be spin up.